shared/js/project.js

```diff
@@ -127,7 +111,7 @@ module.exports = {
       $('.project_updated_at').text("Not saved"); // TODO i18n
     }
   },
-  appToProjectUrl: function () {
+  appToProjectUrl_: function () {
```

Comment:
Good point. If there are cases where we need "this", we could also just pass it in.